Bottom: a22ffa5fb44556409e26b21fe15eeed1ceefa880
Top:    299f0b077ccfe5cef3c491f3be82b01dc6b10a88
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:49:47 -0500

Refresh of asset-host

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 2dfee89d..1a6bd7d1 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -43,7 +43,7 @@
     asset_host: "{{hostvars['nexus']['uri']}}/download"
     buildbot_master: "{{ hostvars['buildbot-master'].ip_addr }}"
     container_etc_hosts:
-      nexus-novatech-llc.com: "{{hostvars['nexus']['ip_addr']}}"
+      nexus.novatech-llc.com: "{{hostvars['nexus']['ip_addr']}}"
     known_hosts: "{{gitlab_keys.stdout}}"
   tasks:
   - name: Scan Gitlab ssh keys
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
index 7dbac6ef..b5c0c70e 100644
--- a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
+++ b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
@@ -36,12 +36,12 @@ PTXCONF_SETUP_SRCDIR="/cache/downloads"
 #
 # PTXCONF_SETUP_NO_DOWNLOAD is not set
 # PTXCONF_SETUP_PTXMIRROR_ONLY is not set
-PTXCONF_SETUP_PTXMIRROR="http://www.pengutronix.de/software/ptxdist/temporary-src"
-PTXCONF_SETUP_DEBMIRROR="http://ftp.uni-kl.de/debian"
-PTXCONF_SETUP_SFMIRROR="http://downloads.sourceforge.net/sourceforge"
-PTXCONF_SETUP_GNUMIRROR="http://ftp.uni-kl.de/pub/gnu"
-PTXCONF_SETUP_XORGMIRROR="http://xorg.mirrors.pair.com/pub/X11/ftp.x.org"
-PTXCONF_SETUP_KERNELMIRROR="http://www.kernel.org/pub/linux"
+PTXCONF_SETUP_PTXMIRROR="http://nexus.novatech-llc.com/repository/orphans"
+PTXCONF_SETUP_DEBMIRROR="http://nexus.novatech-llc.com/repository/debian"
+PTXCONF_SETUP_SFMIRROR="http://nexus.novatech-llc.com/repository/sourceforge"
+PTXCONF_SETUP_GNUMIRROR="http://nexus.novatech-llc.com/repository/gnu"
+PTXCONF_SETUP_XORGMIRROR="http://nexus.novatech-llc.com/repository/xorg"
+PTXCONF_SETUP_KERNELMIRROR="http://nexus.novatech-llc.com/repository/kernel"
 # PTXCONF_SETUP_CHECK_ALWAYS is not set
 PTXCONF_SETUP_CHECK_NOTEMPTY=y
 # PTXCONF_SETUP_CHECK_NEVER is not set
diff --git a/roles/buildbot-worker-ntel/tasks/main.yml b/roles/buildbot-worker-ntel/tasks/main.yml
index 469a03fa..ebe6d2d3 100644
--- a/roles/buildbot-worker-ntel/tasks/main.yml
+++ b/roles/buildbot-worker-ntel/tasks/main.yml
@@ -72,6 +72,7 @@
     tag: '{{ docker_image_tag }}'
     path: '{{ image_dir }}'
     buildargs: '{{ image_args }}'
+    extra_hosts: '{{ container_etc_hosts }}'
     force: true
 
 - name: Start buildbot container
diff --git a/roles/buildbot-worker-ptxdist/defaults/main.yml b/roles/buildbot-worker-ptxdist/defaults/main.yml
index 3dc6fe6c..3fda3523 100644
--- a/roles/buildbot-worker-ptxdist/defaults/main.yml
+++ b/roles/buildbot-worker-ptxdist/defaults/main.yml
@@ -36,6 +36,9 @@ container_env:
 # the name of the container being started
 container_name: '{{ docker_name_prefix }}{{ hostname }}'
 
+# network configuration for the confainer
+container_networks: []
+
 # port configuration of the container
 container_port_args: []
 
diff --git a/roles/buildbot-worker-ptxdist/tasks/main.yml b/roles/buildbot-worker-ptxdist/tasks/main.yml
index 91dec6ce..48c1f82d 100644
--- a/roles/buildbot-worker-ptxdist/tasks/main.yml
+++ b/roles/buildbot-worker-ptxdist/tasks/main.yml
@@ -79,6 +79,7 @@
     tag: '{{ docker_image_tag }}'
     path: '{{ image_dir }}'
     buildargs: '{{ image_args }}'
+    extra_hosts: '{{ container_etc_hosts }}'
     force: true
 
 - name: Start buildbot container
diff --git a/roles/buildbot-worker-ptxdist/templates/pip.conf.j2 b/roles/buildbot-worker-ptxdist/templates/pip.conf.j2
index a54ae09a..2c812530 100644
--- a/roles/buildbot-worker-ptxdist/templates/pip.conf.j2
+++ b/roles/buildbot-worker-ptxdist/templates/pip.conf.j2
@@ -1,3 +1,3 @@
 [global]
-trusted-host = {{hostvars['buildbot-worker-devpi'].uri | urlsplit('hostname')}}
-index-url = {{hostvars['buildbot-worker-devpi'].uri}}
+trusted-host = {{hostvars['nexus'].uri | urlsplit('hostname')}}
+index-url = {{hostvars['nexus'].uri}}/pypi/simple
diff --git a/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2 b/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
index f9fd2372..38e82192 100644
--- a/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
+++ b/roles/buildbot-worker-ptxdist/templates/pydistutils.cfg.j2
@@ -1,2 +1,2 @@
 [easy_install]
-index_url = {{hostvars['buildbot-worker-devpi'].uri}}
+index_url = {{hostvars['nexus'].uri}}/pypi/simple
